contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -284,21 +299,26 @@ contract LidoOracle is ILidoOracle, IsContract, AragonApp {
     function getCurrentReportableEpochs()
         public view
         returns (
-            uint256 firstReportableEpochId,
-            uint256 lastReportableEpochId
+            uint256 minReportableEpochId,
```

Comment:
This will require oracle daemon changes — need not to forget about this.

Reply:
This getter especially is not used by oracle daemon. And other call results are returned as the pythonic list [] (with the default web3.py's ABI decoder) so the names don't really matter here, just position does. And it's unchanged.

Reply:
Ah, got it. `web3.js` returns named vars, too, if the return args are named, so I thought that maybe it's like this on the Python side, too.